### DISCUSSION TITLE
230726 서비스 핵심 가치 이해관계 통일 회의 (10:10~10:20)

### DISCUSSION THREAD

**Discussion by somsom13** [woowacourse-teams/2023-shook#115] [회의]
## 회의 안건
서비스 핵심 가치 이해관계를 통일한다.

**Comment by seokhwan-an** [comment]
우리의 핵심은 유저들이 킬링파트를 만들어 나가면서 즐겁게 소비하는 플랫폼을 만드는 것이다.
그 중에서는 여러 가치가 있는데
- 노래의 킬링파트를 유저들이 만들 수 있다.
- 자신의 파트를 지인들에게 편하게 제공할 수 있다.
- 새로운 노래를 찾는데 킬링파트가 도움이 될 것이다.
